core/src/test/java/org/opensearch/sql/expression/datetime/DateTimeFunctionTest.java

```diff
@@ -614,6 +614,37 @@ public void hour() {
     assertEquals("hour(\"2020-08-17 01:02:03\")", expression.toString());
   }
 
+  public void testInvalidMinuteOfDay(String date) {
```

Comment:
please make this private if they aren't used elsewhere and aren't tests.

Reply:
Fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/189/commits/5778d63fcffc41f5c029f2cbb79bc85250441f7b